desc/compute/_basis_vectors.py

```diff
@@ -1811,59 +1811,43 @@ def _e_sub_rho_rrt(params, transforms, profiles, data, **kwargs):
 def _e_sub_rho_rrz(params, transforms, profiles, data, **kwargs):
     data["e_rho_rrz"] = jnp.array(
         [
-            -2 * data["omega_rz"] * data["R_r"] * data["omega_r"]
```

Comment:
this is the same as the other existing mixed partial; there's nothing to check here just copy pasted